src/app/util/client-api.h

```diff
@@ -94,53 +94,6 @@
 uint16_t emberAfFillExternalBuffer(uint8_t frameControl, chip::ClusterId clusterId, chip::CommandId commandId, const char * format,
```

Comment:
Can't we not delete this?

Reply:
We probably can, but it had a number of callsites in cluster code we compile (messaging-server and ias-zone-server) and I did not want to scope-creep this PR any more than it already had....

But yes, those callsites are obviously buggy and we should do something with them and delete this function.